src/pages/settings/Subscription/SubscriptionPlan.tsx

```diff
@@ -0,0 +1,110 @@
+          ];
+
+    let priceInfo;
+
+    if (isCollect) {
+        priceInfo = isAnnual ? translate('subscription.yourPlan.collect.priceAnnual') : translate('subscription.yourPlan.collect.pricePayPerUse');
+    } else {
+        priceInfo = isAnnual ? translate('subscription.yourPlan.control.priceAnnual') : translate('subscription.yourPlan.control.pricePayPerUse');
+    }
+
+    const onLinkPress = () => {
+        Linking.openURL('https://use.expensify.com/savings-calculator');
+    };
+
+    const isNativeApp = Platform.OS === 'ios' || Platform.OS === 'android';
+
+    return (
+        <Section
+            title={translate('subscription.yourPlan.title')}
+            isCentralPane
+            titleStyles={styles.subscriptionSettingsSectionTitle}
+        >
+            <View style={[styles.subscriptionSettingsBorderWrapper, styles.mt5, styles.p5]}>
+                <Icon
+                    src={isCollect ? Illustrations.Mailbox : Illustrations.ShieldYellow}
+                    width={variables.iconHeader}
+                    height={variables.iconHeader}
+                />
+                <Text style={[styles.yourPlanTitle, styles.mt2]}>{isCollect ? translate('subscription.yourPlan.collect.title') : translate('subscription.yourPlan.control.title')}</Text>
+                <Text style={[styles.yourPlanSubtitle, styles.mb2]}>{priceInfo}</Text>
+                {benefitsList.map((benefit) => (
+                    <View
+                        style={[styles.flexRow, styles.alignItemsCenter, styles.mt2]}
+                        key={benefit}
+                    >
+                        <Icon
+                            src={Expensicons.Checkmark}
+                            fill={theme.iconSuccessFill}
+                        />
+                        <Text style={[styles.yourPlanBenefit, styles.ml2]}>{benefit}</Text>
+                    </View>
+                ))}
+            </View>
+            <View style={[styles.flexRow, styles.alignItemsCenter, styles.mt6]}>
+                <Icon
+                    src={Illustrations.HandCard}
+                    width={variables.iconHeader}
+                    height={variables.iconHeader}
+                    additionalStyles={styles.mr2}
+                />
+                <View style={[styles.flexColumn, styles.justifyContentCenter, styles.flex1, styles.mr2]}>
+                    <Text style={[styles.yourPlanTitle, styles.mt2]}>{translate('subscription.yourPlan.saveWithExpensifyTitle')}</Text>
+                    <Text style={[styles.yourPlanSubtitle, styles.mb2]}>{translate('subscription.yourPlan.saveWithExpensifyDescription')}</Text>
+                </View>
+                {isNativeApp ? null : (
+                    <Button
+                        text={translate('subscription.yourPlan.saveWithExpensifyButton')}
+                        onPress={onLinkPress}
+                        medium
+                    />
+                )}
```

Comment:
```suggestion
                {!isNativeApp && (
                    <Button
                        text={translate('subscription.yourPlan.saveWithExpensifyButton')}
                        onPress={onLinkPress}
                        medium
                    />
                )}
```

Reply:
I think it's not applicable anymore as I moved this code to `<SaveWithExpensifyButton />`